### ISSUE TITLE
Hide Title Bar

### ISSUE THREAD

**Issue by jorge-uptio** [IsmaelMartinez/teams-for-linux#1444]
**Is your feature request related to a problem? Please describe.**
I'm trying to hide the title-bar in liinux, I know it's possible using electron, just can't find it in this app.
I also know there's an older issue on this topic - but doesn't seem to work properly [issue](https://github.com/IsmaelMartinez/teams-for-linux/issues/185)

**Describe the solution you'd like**
Have a flag in the configuration to activate it the "frame: false" option in electron when creating the window

Thanks!

**Comment by IsmaelMartinez** [comment]
Hey, can you find the options in electron? Last time I tried this it was a frameless window (meaning, you only see the app, like a kiosk) that covers the whole screen. 

Things might have changed, as they changed in the past. It used to be possible but then they removed it. 

If you can find more info, I would be happy to check if we can try it out.

**Comment by IsmaelMartinez** [comment]
nope, I was wrong and it was just a bug. Probably when I was doing some refactoring a long time ago. It does do full screen, so not hidding the bar. I will see when I got some time to fix it. Nice catch and thanks for reporting.

**Comment by jorge-uptio** [comment]
Thank you very much!

**Comment by IsmaelMartinez** [comment]
The bug should be fixed in https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.11.2 (pre-release). Can you confirm? Thanks

**Comment by jorge-uptio** [comment]
thanks, it works in full screen mode
I was wondering if there were a solution to remove the title bar when not in full screen mode

anyways, thanks for the hard work

**Comment by IsmaelMartinez** [comment]
I don't think this is possible in linux with electron, sorry. Maybe request this as a feature request in electron.

**Comment by jorge-uptio** [comment]
I believe it is - I'm using owa-desktop and I added the following to the create window:

```javascript
const window = new BrowserWindow({
    width: 1280,
    height: 720,
    title: `Outlook Desktop - ${email}`,
    icon: __dirname + "images/Outlook.ico",
    autoHideMenuBar: true,
    frame: false,
    webPreferences: {
      spellcheck: true,
      webSecurity: true,
      contextIsolation: false,
      webviewTag: true,
      nodeIntegration: true,
      nativeWindowOpen: true,
      session: ses,
      webviewTag: true,
      nodeIntegration: false,
      contextIsolation: true,
    },
  });
```

the `frame: flase`, allowed me to do it - just don't know how to do it in your app.
hope that makes sense

**Comment by jijojosephk** [comment]
@jorge-uptio there is no command-line option in the app. `frame:false` can only be used when creating a new window. So it can be implemented only as a start up parameter.

**Comment by IsmaelMartinez** [comment]
BrowserWindow is deprecated so I will suggest we only try to implement this after we have move away from it to the WebContentView. 

We already got a lot of things breaking from that migration and adding extra changes now will make it move complex. 

I will reopen this but mark it as blocked for those reasons, hope you understand as 3/4 other issues are already block for that migration that will otherwise block future migrations.

**Comment by IsmaelMartinez** [comment]
Ok, I added a `frame` config option that you can try to use. See version https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.11.4

Let us know how you get on. This feature might go away once we move to the WebContentView, but hopefully it doesn't

**Comment by IsmaelMartinez** [comment]
https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.11.5 has this as an optional, not the default. That was a mistake from me.

**Comment by reformat0r** [comment]
This is still broken for me.
Changing this line
https://github.com/IsmaelMartinez/teams-for-linux/blob/c4cfa3e82c9cd4790bf6cb00cc352bba6c1bdbe2/app/mainAppWindow/browserWindowManager.js#L56
to
```js
frame: false
```
does enable the feature.
The problem seems to be type coercion:
```js
typeof this.config.frame ===  "string"
// and
Boolean("false") === true
```

**Comment by IsmaelMartinez** [comment]
Remove the " on your config option and/or use a Falsy value. JavaScript is a bit funny with what is False or True.

**Comment by IsmaelMartinez** [comment]
Well, that is a bug, it should be a boolean like the https://github.com/IsmaelMartinez/teams-for-linux/blob/c4cfa3e82c9cd4790bf6cb00cc352bba6c1bdbe2/app/config/index.js#L219C12-L219C19

**Comment by IsmaelMartinez** [comment]
Ops, forgot to re-open this. 

Just doing a wee release with the fix for this an some small dependencies updates. The type was wrong and changing it to boolean does do as it says in the tin.

Thanks again for reporting.

**Comment by Gounlaf** [comment]
Hi,

This option worked once or twice.... and that's all. I don't why it doesn't work.
I've tried both command line option and files

```
$ teams-for-linux  --logConfig='{"level":"debug"}' --frame=true
Using user configuration
all good with contextIsolation you aren't using them
all good with sandbox you aren't using them
Initialising logger with config: "{\"level\":\"debug\"}"
00:27:15.801 › File logging at /home/gounlaf/.config/teams-for-linux/logs/main.log with level silly
00:27:15.803 › Console logging enabled with level silly
00:27:15.804 › Logger initialised
00:27:15.804 › configPath: /home/gounlaf/.config/teams-for-linux
00:27:15.804 › configFile: { frame: true, menubar: 'visible' }
00:27:15.943 › [CustomNotificationManager] Initialized and listening on "notification-show-toast" channel
00:27:15.945 › (node:62829) [DEP0180] DeprecationWarning: fs.Stats constructor is deprecated.
(Use `teams-for-linux --trace-deprecation ...` to show where the warning was created)
00:27:16.187 › Content Security Policy configured as compensating control for disabled security features
00:27:16.290 › processArgs: [
  '/opt/teams-for-linux/teams-for-linux',
  '--logConfig={"level":"debug"}',
  '--frame=true'
]
00:27:16.292 › testing RegExp processArgs false
00:27:16.293 › testing RegExp processArgs false
00:27:16.293 › testing RegExp processArgs false
00:27:16.295 › Waiting for network...
00:27:16.296 › Testing network using net.request() for https://teams.microsoft.com/v2
00:27:16.301 › Spellchecker is disabled!
00:27:16.305 › [GRAPH_API] IPC handlers registered { channels: 5, enabled: false }
00:27:16.307 › [GLOBAL_SHORTCUTS] No global shortcuts configured
00:27:16.307 › 🔒 IPC Security: Channel allowlisting enabled
00:27:16.307 › 🔒 IPC Security: 43 channels allowlisted
00:27:16.831 › Network test successful with method https
00:27:16.831 › Loading initial URL...
00:27:18.351 › did-frame-finish-load {} true
00:27:18.352 › did-finish-load
00:27:21.485 › Badge count set to '0'
00:27:21.499 › Badge count set to '0'
00:27:21.509 › Badge count set to '0'
00:27:21.511 › Badge count set to '0'
00:27:23.865 › Badge count set to '0'
00:27:23.866 › Badge count set to '0'
00:27:27.650 › did-frame-finish-load {} false
00:27:29.793 › Badge count set to '0'
00:27:29.795 › Badge count set to '0'
00:27:29.896 › did-frame-finish-load {} false
00:27:30.541 › did-frame-finish-load {} false
```

- Ubuntu 24.04
- TFL installed via APT repositories
- Windowing System: X11

Please tell me what/if I can help (more logs, tests...).

Maybe it's a bug from Electron side?

Thanks.

**Comment by IsmaelMartinez** [comment]
it should be frame -> false. Frame true is the default value (to have a frame). Thanks

**Comment by Gounlaf** [comment]
Hi @IsmaelMartinez,

I want a frame! But neither true or false works.

**Comment by IsmaelMartinez** [comment]
oh, that is probably your window manager playing with you. I can see it with and without the frame, but I do use macOS most of the time
